openfisca_uk/variables/person/disability.py

```diff
@@ -119,7 +119,7 @@ def formula(person, period, parameters):
             # Given to a single person at benunit level
             "PIP_DL",
             "PIP_M",
-            # ESA_income omitted since it is reported at benunit level
+            "ESA_income",
```

Comment:
```suggestion
            "ESA_income_reported_personal",
```

In the PR (#68)  adding the personal ESA income benefit, I kept the original ```ESA_income```, and just added the personal reported amount, ```ESA_income_reported_personal```, so we'd probably want to use this?